src/pages/home/report/ReportActionCompose/SuggestionMention.tsx

```diff
@@ -83,17 +106,25 @@ function SuggestionMention(
         [currentUserPersonalDetails.login],
     );
 
+    const getMentionCode = useCallback(
+        (mention: Mention, mentionType: string): string => {
+            if (mentionType === '#') {
+                // room mention case
+                return mention.login ?? '';
```

Comment:
Before Mention type was only used for users, thats why key names are named with users in mind. I didn't change it, because I didn't want to make this PR bigger. So I just follow the `Mention` type - in the line 264 we are setting login value to roomName.

LEt me know If I should change the `Mention` type naming convencion so it would be more universal

Reply:
Ahh I see. I feel like we should change it. The way it is right now feels confusing to me. What do you think @rlinoz?

Reply:
I missed this one, sorry.

Yeah, I agree we should change it, `mention.handle` maybe?

Reply:
Yeah I like that. Or maybe `mention.displayText`

Reply:
On it 👀

Reply:
@puneetlath @rlinoz Went with `handle` as with `displayText` we would end up with three different fields named around text: `text`, `alternateText`, `displayText`